src/query/storages/fuse/src/operations/replace_into/mutator/mutator_replace_into.rs

```diff
@@ -62,13 +62,19 @@ impl ReplaceIntoMutator {
     ) -> Result<MergeIntoOperation> {
         let num_rows = data_block.num_rows();
         let mut columns = Vec::with_capacity(self.on_conflict_fields.len());
+        let data_block = data_block.convert_to_full();
```

Comment:
Why not change `build_column_hash`  to be :

```
fn build_column_hash(
        columns: &[&ValueRef<AnyType>],
        saw: &mut HashSet<UniqueKeyDigest>,
        num_rows: usize,
    ) -> Result<ColumnHash> 
```

Reply:
Yes, you can take the Scalar value if it's ValueRef::Scalar for `columns_min_max ` with single match